no_mutations.md

```diff
@@ -11,20 +11,266 @@ kernelspec:
   name: python3
 ---
 
+```{currentmodule} tskit
+```
+
 (sec_tskit_no_mutations)=
 
-# _Do you really need mutations?_
-% remove underscores in title when tutorial is complete or near-complete
+# Do you really need mutations?
+
+In tree sequences, the genetic genealogy exists independently of the mutations that
+generate genetic variation. This tutorial aims to illustrate
+cases where the genealogy on its own may be sufficent; a number of
+analyses do not actually require genetic variation.
+
+This applies particularly to cases where you generate tree sequences by simulation.
+In this case, although simulating variable sites allows you to generate actual genetic
+data, it's worth considering the following points:
+
+1. Neutral mutations and sites can be added to a genealogy later, if you find they *are*
+needed
+2. Simulating sites and mutations increases memory requirements and takes marginally
+more CPU time
+3. Storing mutations increases the tree sequence file size
+4. Some genetic analyses can be performed equally well (or better!) on tree sequences
+that have no sites or mutations.
```

Comment:
```suggestion
3. Quantities of interest can often be inferred equally well (or better!) on tree sequences
that have no sites or mutations.
```
Changing this because saying "genetic analyses" really sounds like you're using real data, where you definitely do have mutations.

Reply:
Good point